packages/addon-dev/package.json

```diff
@@ -42,7 +42,7 @@
     "@types/fs-extra": "^9.0.12",
     "@types/minimatch": "^3.0.4",
     "@types/yargs": "^17.0.3",
-    "rollup": "^2.58.0",
+    "rollup": "^3.23.0",
```

Comment:
See https://github.com/rollup/rollup/releases/tag/v3.0.0 for breaking changes that might affect `@embroider`.

Reply:
@NullVoxPopuli Can you run CI for me?